include/bm/bm_sim/P4Objects.h

```diff
@@ -378,6 +380,10 @@ class P4Objects {
   void init_meter_arrays(const Json::Value &root, InitState *);
   void init_register_arrays(const Json::Value &root);
   void init_actions(const Json::Value &root);
+  void check_next_nodes(const Json::Value &cfg_next_nodes,
+                        const Json::Value &cfg_actions,
+                        const string table_name,
```

Comment:
no reason to pass the string by value here I think, so ideally this would be `const std::string &table_name`

it would be more consistent with other methods as well

Reply:
updated in commit 4